main.py

```diff
@@ -25,7 +25,7 @@
     TokenUsage,
 )
 
-DEFAULT_MODEL = "THUDM/chatglm-6b-int4"
+DEFAULT_MODEL = "bigscience/bloomz-560m"
```

Comment:
Do we support this in LLMSpec?

Reply:
Yes. But I didn't test the multi-round conversations.

Reply:
https://github.com/tensorchord/llmspec/blob/main/llmspec/llmspec.py#L81

We do not set the token for it